detection-rules/impersonation_github_sawfish.yml

```
@@ -12,8 +12,8 @@ source: |
     or headers.return_path.domain.root_domain != 'github.com'
   )
   and 2 of (
-    iregex_search(body.plain.raw, '.*account activity.*', '.*github.*', '.*your activity.*', '.*suspicious api call.*'),
-    iregex_search(body.html.raw, '.*account activity.*', '.*github.*', '.*your activity.*', '.*suspicious api call.*'),
+    iregex_search(body.plain.raw, '.*account activity.*', '.*your activity.*', '.*suspicious api call.*'),
```

btw, these are all additional ways to write the same thing:
```suggestion
    ilike(body.plain.raw, '*account activity*', '*your activity*', '*suspicious api call*'),
```

```suggestion
    iregex_search(body.plain.raw, 'account activity', 'your activity', 'suspicious api call'),
```

```suggestion
    iregex_match(body.plain.raw, '.*account activity.*', '.*your activity.*', '.*suspicious api call.*'),
```